lib/generators/lightning_cad/test_generator.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails'
+
+module LightningCad
+  class TestGenerator < Rails::Generators::Base
+    source_root File.expand_path('./templates', __dir__)
+
+    def add_jasmine_tests
+      say 'Adding jasmine'
+      run "yarn add --dev jasmine@^5.1.0"
+
+      run 'npm pkg set scripts.test_shared="NODE_ENV=test NODE_PATH="./node_modules:./app/javascript:$NODE_PATH" jasmine --config=jasmine.json"'
```

Comment:
I don't think NODE_PATH works with ESM, but we can add a custom import hook to replicate the behavior.